content/docs/command-reference/plots/index.md

```diff
@@ -12,46 +12,37 @@ usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
 
 positional arguments:
   COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+    show         Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff         Show multiple versions of plot data by plotting them in a single image.
+    modify       Modify display properties of data-series plot outputs (has no effect on image-type plots).
 ```
 
-## Types of metrics
-
-DVC has two concepts for metrics, that represent different results of machine
-learning training or data processing:
-
-1. `dvc metrics` represent **scalar numbers** such as AUC, _true positive rate_,
-   etc.
-2. `dvc plots` can be used to visualize **data series** such as AUC curves, loss
-   functions, confusion matrices, etc.
-
 ## Description
 
-DVC provides a set of commands to visualize certain metrics of machine learning
-experiments as plots. Usual plot examples are AUC curves, loss functions,
-confusion matrices, among others.
+DVC provides a set of commands to visualize data produced by machine learning
+experiments. Usual plot examples are AUC curves, loss functions, confusion
+matrices, among others.
 
-This type of metrics files are created by users, or generated by user data
-processing code, and can be defined in `dvc.yaml` (`plots` field) for tracking
-(optional).
+This type of data is created by users, or generated by user data processing
+code.
 
-DVC can work with two types of plots files:
+## Types of plots
+
+`dvc plots` is able to visualize two types of data:
```

Comment:
I'd roll this change back, or at least avoid adding more structure. The **Description** should not need more H2/3s. If it does it's a sign that we need a new User Guide on top of this change...